benchmarks/src/main/java/org/questdb/LineTCPSenderMainVarLenStrings.java

```diff
@@ -90,26 +96,42 @@ private static CharSequence nextString(int len, Rnd rnd) {
         return sink;
     }
 
-    private static class LoggingLineTcpSender extends LineTcpSender {
+    private static class LoggingLineChannel implements LineChannel {
```

Comment:
this is just adapting an existing test. in actually illustrates the new `LineChannel` abstraction quite nicely: When you want to log network traffic to a file you don't have to extend `LineTcpSender` where is a lot of complexity. Instead, you just implement a simple  `LineChannel` and pass it to the `LineTcpSender`